2.3/Dockerfile.rhel7

```diff
@@ -0,0 +1,41 @@
+FROM rhscl/s2i-base-rhel7
```

Comment:
not sure how this passed tests since this image doesn't exist?  i'm going to do a PR to change it to openshift/base-rhel7 like the others.